### DISCUSSION TITLE
Log files location

### DISCUSSION THREAD

**Discussion by TheRealUmbray** [Rem0o/FanControl.Releases#2005] [Q&A] [answered]
Hi everyone

Whenever fancontrol starts I get the following message "Logs have grown to 3 files. Consider checking them out and deleting them"

However, I have been unable to locate the log files, so I do not know whether they are error logs or simply log files that are starting to get large.

Anyone able to help out?

**Comment by Rem0o** [comment, marked as answer]
log files will be next to your fan control executable.
